components/eventing-controller/README.md

```diff
@@ -70,7 +70,7 @@ This section explains how to use the Eventing Controller. It expects the followi
 |  `JS_STREAM_RETENTION_POLICY`     | The policy to delete events from the stream: `limits` or `interest`. See https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive#stream-limits-retention-and-policy. |
 |  `JS_STREAM_MAX_MSGS`             | The maximum number of messages in the stream. Used only when storage policy is set to `limits`. |
 |  `JS_STREAM_MAX_BYTES`            | The maximum size of the stream in bytes. Used only when storage policy is set to `limits`.     |
-|  `JS_CONSUMER_DELIVER_POLICY`     | The policy to deliver events to consumers from the stream. Supported values are: `all`, `last`, `last_per_subject`, and `new`. See https://docs.nats.io/nats-concepts/jetstream/consumers#deliverpolicy-optstartseq-optstarttime. |
+|  `JS_CONSUMER_DELIVER_POLICY`     | The policy to deliver events to consumers from the stream. Supported values are: `all`, `last`, `last_per_subject`, and `new`. See https://docs.nats.io/nats-concepts/jetstream/consumers#deliverpolicy. |
```

Comment:
I have some doubts about it.
The values in the linked documentation are different that the allowed values given in this table. 
 Has this been consulted with the Eventing team?

Reply:
My guess was that nats simply removed an anchor from their page and this seemed like the best match for replacement. 

I'll ask the Eventing team for verification. Thanks for spotting this!

Reply:
@NHingerl your link is correct. Thanks for checking and updating!